ui/components/multichain/import-nfts-modal/import-nfts-modal.js

```diff
@@ -77,13 +97,24 @@ export const ImportNftsModal = ({ onClose }) => {
   const [nftAddFailed, setNftAddFailed] = useState(false);
   const trackEvent = useContext(MetaMetricsContext);
 
+  const [actionMode, setActionMode] = useState(ACTION_MODES.IMPORT_NFT);
+
+  const [selectedNetworkForCustomImport, setSelectedNetworkForCustomImport] =
+    useState(null);
+
+  const allNetworks = useSelector(getNetworkConfigurationsByChainId);
+  const networkConfigurations = useSelector(getNetworkConfigurationsByChainId);
+  const networkClientId = useSelector(getSelectedNetworkClientId);
```

Comment:
we are passing this to the NftController which is returning the selectedNetworkClientId but it should instead be the networkClientId that the user chose from the form 🙏

Reply:
Fixed here: https://github.com/MetaMask/metamask-extension/pull/32020/commits/d985f0c338f9c6644491e4de20f8269192ba2945 will add some tests here as well before merging